Callback/SMSFactorCallback.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace MauticPlugin\MauticSmsFactorBundle\Callback;
+namespace MauticPlugin\MauticSMSFactorBundle\Callback;
```

Comment:
I would keep the previous naming, since it could cause some troubles with conventions based on the name conversion (camelCase/PascalCase to snake_case ➜ mautic_s_m_s_factor_bundle